instant-xml/src/impls.rs

```diff
@@ -130,24 +114,9 @@ impl ToXml for char {
     where
         W: fmt::Write,
     {
-        match serializer.consume_field_context() {
-            Some(field_context) => {
-                let mut tmp = [0u8; 4];
-                let char_str = self.encode_utf8(&mut tmp);
-                match field_context.attribute {
-                    Some(FieldAttribute::Attribute) => {
-                        serializer.add_attribute_value(char_str);
-                    }
-                    _ => {
-                        serializer.add_open_tag(&field_context)?;
-                        serializer.output.write_str(char_str)?;
-                        serializer.add_close_tag(field_context)?;
-                    }
-                }
-                Ok(())
-            }
-            None => Err(Error::UnexpectedValue),
-        }
+        let mut tmp = [0u8; 4];
+        let converted = special_xml_entities_handling(&&*self.encode_utf8(&mut tmp));
```

Comment:
Should deal with the lint warning here.

Reply:
What lint do you use? Clippy does not show me anything here. I'm using: `cargo clippy --workspace --all-targets --all-features -- -D warnings`

And this is changed already.